src/Views/OperatingSystemView.vala

```diff
@@ -125,17 +125,24 @@ public class About.OperatingSystemView : Gtk.Grid {
 
         var settings_restore_button = new Gtk.Button.with_label (_("Restore Default Settings"));
 
+        var reboot_to_firmware_setup_button = new Gtk.Button.with_label (_("Reboot to Firmware Setup"));
```

Comment:
It looks like in the session indicator we're using "Restart" instead of "Reboot"
```suggestion
        var reboot_to_firmware_setup_button = new Gtk.Button.with_label (_("Restart to Firmware Setup"));
```